style.css

```diff
@@ -359,3 +143,479 @@ form {
+    padding-bottom: 142px;
+    padding-top: 76px;
+  }
+
+  .card {
+    display: flex;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    margin-top: 142px;
+    border-radius: 24px;
+    padding: 24px;
+  }
+
+  .card1 {
+    margin-top: 0;
+  }
+
+  .profile-image {
+    flex: 1;
+    width: 544px;
+    height: 448px;
+  }
+
+  .card-item {
+    flex: 1;
+    margin-left: 10px;
+  }
+
+  .btn-enabled {
+    margin-top: 54px;
+  }
+
+  .canopy {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .bg {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 24px;
+    color: #7a869a;
+  }
+
+  .counter {
+    width: 8px;
+    height: 8px;
+    margin: -1px -1px;
+  }
+
+  #about {
+    background-image: url(images/about-container.jpeg);
+    display: flex;
+    padding: 142px 182px 154px 142px;
+    border-top-right-radius: 50px;
+    border: 1px #fff solid;
+  }
+
+  .about-item1 {
+    flex: 1;
+  }
+
+  .about-item2 {
+    flex: 1;
+  }
+
+  .lang {
+    display: flex;
+  }
+
+  .lang-text {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    align-items: center;
+    color: #344563;
+  }
+
+  .lang-drop-down {
+    margin-left: auto;
+  }
+
+  .lang-list {
+    display: flex;
+    list-style-type: none;
+    padding: 10px;
+  }
+
+  .lang-list-box {
+    width: 122px;
+    height: 120px;
+    left: 12px;
+    top: 12px;
+    background: #f7f7f9;
+    padding: 12px;
+    border-radius: 8px;
+    margin-right: 12px;
+  }
+
+  .lang-list-box img {
+    padding-bottom: 16px;
+  }
+
+  hr {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    background: #dfe1e6;
+  }
+
+  .mt {
+    margin-top: 36px;
+  }
+
+  footer {
+    background-image: url(images/ContactMe.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-wrap: wrap;
+    width: 1440px;
+    height: 846px;
+    justify-content: center;
+    align-content: center;
+  }
+
+  .contact-me {
+    width: 801px;
+    text-align: center;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+  }
+
+  .contact-text {
+    text-align: center;
+    width: 801px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    color: #ebebff;
+    padding-right: 300px;
+    padding-left: 300px;
+  }
+
+  form {
+    width: 801px;
+  }
+
+  .input-container {
+    margin-top: 20px;
+    display: flex;
+    padding-right: 160px;
+    padding-left: 160px;
+  }
+
+  .input {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding: 15px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 20px;
+    display: flex;
+    color: #172b4d;
+    width: 400px;
+  }
+
+  .btn-submit-container {
+    padding-right: 300px;
+    padding-left: 300px;
+  }
+}
```

Comment:
- [ ] [REQUIRED] You have done a great job by setting the minimum width of your screen to 992px, however, you have horizontal and vertical scroll bars. Please note that you are not supposed to have this. I inspected your design with the chrome dev tools and it seems way bigger. 
 I would recommend making the width 100% and don't fix it so your design can fit into different desktop screen sizes.. Well done 👍🏽 
![desktop](https://user-images.githubusercontent.com/58692590/149401748-2d74ab59-bd17-4e8c-be10-7917e00a9042.png)